lib/api/definitions/__tests__/MessageBodyMarkdown.test.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  isMessageBodyMarkdown,
+  toMessageBodyMarkdown
+} from "../MessageBodyMarkdown";
+
+import { toWithinRangeString, WithinRangeString } from "../../../utils/strings";
+
+import { Option } from "ts-option";
+
+describe("Check MessageBodyMarkdown type", () => {
+  test("MessageBodyMarkdown string within range (80-10000)", () => {
+    // String long 90 characters.
+    const text: string =
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt";
+    const t: Option<WithinRangeString<80, 10000>> = toWithinRangeString(
+      text,
+      80,
+      10000
+    );
+
+    expect(toMessageBodyMarkdown(text)).toEqual(t);
+  });
+
+  test("isMessageBodyMarkdown on string within range borders (80-10000 characters)", () => {
+    const text: string =
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt";
+
+    expect(isMessageBodyMarkdown(text)).toBe(true);
+  });
+
+  test("isMessageBodyMarkdown on string shorter then 80 characters", () => {
+    // String shorter than 80 characters.
+    const text: string = "Lorem ipsum dolor sit amet";
+    expect(isMessageBodyMarkdown(text)).toBe(false);
+  });
+
+  test("isMessageBodyMarkdown on string longer then 10000 characters", () => {
+    // String long 90 characters.
+    const text: string =
```

Comment:
maybe we should consider adding a libray like faker to generate fixtures
https://github.com/marak/Faker.js/